KsApi/extensions/ApolloClient+Singleton.swift

```diff
@@ -25,7 +25,7 @@ class GraphQL {
   func configure(
     with url: URL,
     headers: [String: String],
-    additionalHeaders: @escaping () -> [String: String]
+    additionalHeaders: [String: String]
```

Comment:
is there a reason you removed @escaping here? we are relying on a network request to get these headers so it's probably a good idea to keep it.

Reply:
Most of the changes below seem to be a result of this so I'm trying to be careful about the specifics here.